ocrd_utils/ocrd_logging.conf

```diff
@@ -54,28 +54,23 @@ handlers=consoleHandler
 #qualname=ocrd.workspace
 
 # ocrd loggers
-[logger_ocrd_ocrd]
+[logger_ocrd]
 level=ERROR
 handlers=consoleHandler
 qualname=ocrd
+propagate=0
 
-[logger_ocrd_ocrd_models]
+[logger_ocrd_models]
 level=INFO
 handlers=consoleHandler
 qualname=ocrd_models
-[logger_ocrd_ocrd_utils]
-level=INFO
-handlers=consoleHandler
-qualname=ocrd_utils
-[logger_ocrd_ocrd_network]
-level=INFO
-handlers=consoleHandler
-qualname=ocrd_network
-[logger_ocrd_ocrd_exif]
-level=INFO
-handlers=consoleHandler
-qualname=ocrd_exif
```

Comment:
Why is `ocrd_models` still involved? Isn't this already handled by `logger_ocrd` above?

Reply:
It was to be removed in the the part of #1101 that I had to abandon because of the bizarre test failures. In fact that was the point of that PR, to make all the loggers consistently inherit from `ocrd`. I will still do that, but let's first finish this and #1083.

Reply:
`ocrd_models` is gone now.